lib/LANraragi/Plugin/Metadata/RegexParse.pm

```diff
@@ -23,34 +74,49 @@ sub plugin_info {
         type        => "metadata",
         namespace   => "regexplugin",
         author      => "Difegue",
-        version     => "1.0",
+        version     => "1.0.1",
         description =>
           "Derive tags from the filename of the given archive. <br>Follows the doujinshi naming standard (Release) [Artist] TITLE (Series) [Language].",
         icon =>
           "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAAXNSR0IArs4c6QAAAL1JREFUOI1jZMABpNbH/sclx8DAwPAscDEjNnEMQUIGETIYhUOqYdgMhTPINQzdUEZqGIZsKBM1DEIGTOiuexqwCKdidDl0vtT62P9kuZCJEWuKYWBgYGBgRHbh04BFDNIb4jAUbbSrZTARUkURg6lD10OUC/0PNaMYgs1Skgwk1jCSDCQWoBg46dYmhite0+D8pwGLCMY6uotRDOy8toZBkI2HIhcO/pxCm8KBUkOxFl/kGoq3gCXFYFxVAACeoU/8xSNybwAAAABJRU5ErkJggg==",
-        parameters => [ ]
+        parameters => [
+            { type => "bool", desc => "Capture trailing tags in curly brackets" },
+            {   type => "bool",
+                desc => "Keep everything you catch as tags in the namespace \"${PLUGIN_TAG_NS}\"<BR />"
```

Comment:
It's unclear for end users what will or will not end up with `parsed:` prepended to it with or without this parameter enabled. 

From what I can gather:  
- specific "common extraneous values" will always get the `parsed:` namespace  
- languages we couldn't figure out will always get the `parsed:` namespace (I think this should be changed as per my comment above)  
- trailing tags in curly brackets will get the `parsed:` namespace if this option is enabled  

Is that correct? I'm not sure how to explain it better atm but I can try thinking about a better description.

Reply:
What I noticed is that there are often useful categorizations that are found beyond the standard part or even that if the standard part does not exist, especially for different media, there can still be a pattern traceable between similar media. So it could be convenient for the advanced user if we extract these labels making them manageable with the rules.

By default the plugin doesn't do anything different from before, but by enabling the advanced options the user can use the namespace `parsed:` to easily identify and rewrite the labels that can be of interest by creating some ad hoc tag rules.
What remains can be cleaned up by putting `-parsed:*` as the last rule.